hyperlink/_url.py

```diff
@@ -907,7 +888,6 @@ def replace(self, scheme=_UNSET, host=_UNSET, path=_UNSET, query=_UNSET,
             port=_optional(port, self.port),
             rooted=_optional(rooted, self.rooted),
             userinfo=_optional(userinfo, self.userinfo),
-            family=_optional(family, self.family),
```

Comment:
And, further, upon remembering that `replace` never even had it, I suppose I could be convinced that this is an outlier sufficiently extreme in its low level of usefulness and short lifetime that it's not worth the additional effort to deprecate.  (I really wish github had a code search which could help open source maintainers with this kind of thing though.)